aggregations/db_tables/daily_accounts_added_per_entity.py

```diff
@@ -0,0 +1,98 @@
+from . import DAY_LEN_SECONDS, daily_start_of_range
+from ..periodic_aggregations import PeriodicAggregations
+
+"""
+This code builds a table intended to assist with calculating statistics
+relating accounts ("added_accounts") to ecosystem entities ("entities")
+and is originally intended to be used for tracking new accounts added to each entity.
+"""
+
+
+class DailyAccountsAddedPerEntity(PeriodicAggregations):
```

Comment:
I am sorry we spend so much time on naming, but I believe it is better to spend a week on it once instead of explaining the confused users what it means for years after.

I feel that "Entries" in the general context of near-analytics is not specific enough. Let's add "Ecosystem" to it, so we will have `DailyAccountsAddedPerEcosystemEntry` (and the table should also be renamed)

Reply:
The naming is currently `Entity` not `Entry`, as per the language used by the Ecosystem team. I do think its a good idea to add "ecosystem" into the naming, i just wanted to double check that `DailyAccountsAddedPerEcosystemEntity` is suitable before making the change